android/titanium/src/java/ti/modules/titanium/TitaniumModule.java

```diff
@@ -274,11 +274,21 @@ public String stringFormatDate(Date date, @Kroll.argument(optional=true) String
 	}
 
 	@Kroll.method @Kroll.topLevel("String.formatTime")
-	public String stringFormatTime(Date time)
+	public String stringFormatTime(Object time)
 	{
 		int style = DateFormat.SHORT;
 
-		return (DateFormat.getTimeInstance(style)).format(time);
+		if (time != null && time instanceof Date) {
```

Comment:
You don't need to do a `null` check. It's safe to pass a `null` reference to `instanceof` and it'll return false in that case. So, change to this...

```java
if (time instanceof Date) {
```